src/connectors/emby.js

```diff
@@ -1,37 +1,56 @@
 'use strict';
 
-const trackArtSelector = '.nowPlayingBarInfoContainer .nowPlayingImage';
+Connector.trackArtSelector = [
+	// Emby v4.6.7
+	'.nowPlayingBarInfoContainer .nowPlayingBarImage',
+	'.nowPlayingBarInfoContainer .nowPlayingImage'
+];
 
 Connector.playerSelector = '.nowPlayingBar';
 
-Connector.artistSelector = '.nowPlayingBarText .nowPlayingBarSecondaryText';
+Connector.artistSelector = [
+	// Emby v4.6.7
+	'.nowPlayingBarText .secondaryText',
+	'.nowPlayingBarText .nowPlayingBarSecondaryText'
+];
 
 Connector.trackSelector = [
 	// Jellyfin v10.6.0 and newer
-	'.nowPlayingBarText div:nth-child(1) a',
+	'.nowPlayingBarText div:nth-child(1)',
 	// Emby or Jellyfin v10.5.0 and older
 	'.nowPlayingBarText .textActionButton',
 ];
 
-Connector.trackArtSelector = trackArtSelector;
+// Connector.albumSelector = [
```

Comment:
Should this be commented out?

Reply:
Should I remove this comment block? I added the album selector code but it does not work well because element not refreshed when invisible. I will also fix other styling issues.

Reply:
Sorry missed this was out for a while - best to remove code that's not used to avoid confusion.

Aside this + lint issues looks good :+1: